.circleci/config.yml

```diff
@@ -2,23 +2,23 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: cimg/node:lts
+      - image: cimg/node:16.16
```

Comment:
Why Node 16 and not LTS?

Reply:
The build process requires use of a legacy SSL algo mentioned in the PR description; it's just a coin flip as to whether to use Node 16 or add the node options flag, I'll make the change to add the flag

Reply:
Fixed